src/pet.js

```diff
@@ -0,0 +1,59 @@
+const MAXIMUM_FITNESS = 10;
+const AGE_AT_BIRTH = 0;
+const NOT_HUNGRY = 0;
+const isDeadError = new Error('Sorry, your pet has gone to live on the big farm in the sky');
+
+function Pet(name) {
+    this.name = name,
+    this.age = AGE_AT_BIRTH,
+    this.hunger = NOT_HUNGRY,
+    this.fitness = MAXIMUM_FITNESS,
+    this.children = []
+};
+
+Pet.prototype = {
+    get isAlive() {
+        return this.fitness > 0 && this.hunger < 10 && this.age < 30
```

Comment:
why not use constants to replace these magic numbers? that way it's more descriptive of what these values actually refer to and will make it easier for someone who's unfamiliar with the project to understand